src/components/Cards/CardLarge/CardLarge.scss

```diff
@@ -0,0 +1,68 @@
+@import './../../../styles/utilities/colors.scss';
+@import './../../../styles/fonts.scss';
+@import './../../../styles/utilities/typography.scss';
+
+$card-image-height: 330px;
+
+/* stylelint-disable */
+.card-large {
+  display: grid;
+  max-width: 600px;
+  min-height: 610px;
+  background: $light-grey;
+  border-radius: 5px;
+  box-shadow: 0px 2px 1px rgba(0, 0, 0, 0.14), 0px 4px 5px rgba(0, 0, 0, 0.12);
+  grid-template-areas: 
+    "image"
+    "text"
+    "button";
+  grid-template-rows: $card-image-height auto 74px;
+  text-align: center;
+  overflow: hidden;
+  justify-self: center;
+}
+/* stylelint-enable */
+
+.card-large__image {
```

Comment:
Please style in BEM styling. See DemoPage comment for example.

Reply:
added BEM styling in latest push.